### DISCUSSION TITLE
add a cool down before searching in song select

### DISCUSSION THREAD

**Discussion by The-Grace-God** [ppy/osu#25462] [Ideas]
When going back to song select add a cool down before you can type anything into the search bar as often times I will go back to the song select and accidentally type something and lose where I was at

**Comment by peppy** [comment]
Can you give an example of what you're accidentally typing?

**Reply by The-Grace-God** [reply]
When quickly exiting out of a song by using retry and escape basically ~ then esc occasionally I'll accidentally hit the 1 key or ~ again And completely lose the spot I was in the in the song list

**Reply by verarr** [reply]
There is a cooldown, but it feels too short. (as in, I think I remember stable having a longer cooldown.)

For example, I'd be typing filters like `status=r` but I'm not quick enough to hit the _equals sign_ so it'll just search for `status` which probably isn't the map I currently had selected.
